Create themes

shushugah opened the pull request:
Some open questions remain. Currently the url structure goes as follows: 

- [/themes](https://6067a2b2cf56ce0008550c61--techworkersberlin.netlify.app/themes) # links to the following, but doesn't change their url necessarily  
   - /news
   - /works-councils 
   -  /learning 
   - [/sectors](https://6067a2b2cf56ce0008550c61--techworkersberlin.netlify.app/sectors)
     - [/sectors/amazon](https://6067a2b2cf56ce0008550c61--techworkersberlin.netlify.app/sectors/amazon) #note how Events/articles are dynamically queried. So whenever we add new events, these will be auto-populated.
     - /sectors/gaming 
     - [/sectors/gentrification ](https://6067a2b2cf56ce0008550c61--techworkersberlin.netlify.app/sectors/gentrification)

Some open questions remain: 

Do we want sectors to be a high-level theme, along with other high-level themes like "press", "migration", "works councils"? Should the URL's all reflect these hierarchies? e.g. `/themes/sectors/amazon` vs `/sectors/amazon` vs `/amazon`. All are technically possible and accidental name colision is unlikely.

shushugah committed:
Add custom match_tag_filter plugin, which accepts a string or an array of tags and filters collection of posts. If there are no filters (default) then it returns all of the posts. This allows for /learning to be nearly deduplicated

shushugah committed:
Add theme concept

chobeat commented:
Personally I would go for the second

mleimeister commented:
Hi, that looks great! I'd also favor the `/sectors/amazon` I think, as it groups the sectoral sub-sites nicely together (compared to just `/amazon`), and in the first option, the `/themes/*` root  seems redundant, as most other high-level sites are also some sort of a theme, right?

shushugah committed:
Add hover hamburger menu

shushugah commented on _sass/_header.scss:
```diff
@@ -131,3 +131,45 @@ header {
+  right: 0;
+}
+
+.dropdown__header .dropdown__category:hover .dropdown__menu {
```
the hover category here is likely too small, should more padding be added to `.dropdown__category` or should the hover pseudo selector be added elsewhere? @ovlb any ideas?

shushugah committed:
Replace CSS hover solution with javascript toggle class

standardize color (different shades of red used) but mostly replace hex values with existing variables

shushugah committed:
Update about page and add sketch note
